frontend_new/src/components/Cards.tsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 275,
+  },
+  bullet: {
+    display: 'inline-block',
+    margin: '0 2px',
+    transform: 'scale(0.8)',
+  },
+  
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    marginBottom: 12,
+  },
+});
+
+type props = {
+    primary : string;
+    secondary: string;
+  };
+  
+export default function SimpleCard(prop: props) {
+  const classes = useStyles();
+  const bull = <span className={classes.bullet}>•</span>;
+
+  return (
+    <Card className={classes.root}>
+      <CardContent style={{backgroundColor: "#F3F3F1"}}>
+        <Typography variant="h5" component="h2">
```

Comment:
**[Question, Partial API Usage]** Is there a reason that we are using a header with a raw `h2` component as opposed to the `CardHeader` component intended for cards? Seems like we should have subclassed styling for this particular variant of a card with the header if we want a custom header style.